x-pack/plugins/security_solution/public/management/components/console/components/console_manager/console_manager.tsx

```diff
@@ -14,7 +14,8 @@ import React, {
   useRef,
   useState,
 } from 'react';
-import { ConsolePopup } from './components/console_popup';
+import { ConsoleDataState } from '../console_state/types';
```

Comment:
import as `type`

Reply:
I wonder why ESLint did not catch this here, but does in other areas of Security Solution. I'll have to look into that.